react/features/face-landmarks/FaceLandmarksHelper.ts

```diff
@@ -59,11 +61,12 @@ export class HumanHelper implements FaceLandmarksHelper {
         deallocate: true,
         filter: { enabled: false },
         face: {
-            enabled: true,
+            enabled: false,
```

Comment:
Is this intentional?

Reply:
It will be set to true in the init function of the helper